libs/client/dashboard/src/lib/components/tags-management/child-tag-item/child-tag-item.component.ts

```diff
@@ -1,19 +1,19 @@
 import { Component, Input } from '@angular/core';
-import { ChildTagsModel } from '@dragonfish/shared/models/content';
 import { TagsService } from '@dragonfish/client/repository/dashboard/tags';
 import { AlertsService } from '@dragonfish/client/alerts';
 import { MatDialog } from '@angular/material/dialog';
 import { ChildTagFormComponent } from '../child-tag-form/child-tag-form.component';
 import { PopupModel } from '@dragonfish/shared/models/util';
 import { PopupComponent } from '@dragonfish/client/ui';
+import { TagsModel } from '@dragonfish/shared/models/content';
 
 @Component({
     selector: 'dragonfish-child-tag-item',
     templateUrl: './child-tag-item.component.html',
 })
 export class ChildTagItemComponent {
```

Comment:
Should this component exist at all, given that the concept of a child tag doesn't really exist anymore? (At least not as a unique thing--any tag can conceivably be a child OR a parent tag)